jung-visualization/src/main/java/edu/uci/ics/jung/visualization/BasicVisualizationServer.java

```diff
@@ -173,6 +154,11 @@ public BasicVisualizationServer(
     renderingHints.put(RenderingHints.KEY_ANTIALIASING, RenderingHints.VALUE_ANTIALIAS_ON);
 
     renderContext.getMultiLayerTransformer().addChangeListener(this);
+    try {
+      VisualizationViewerUI.getInstance(this).parse();
+    } catch (IOException e) {
+      log.debug("Unable to read property files. Using defaults.");
+    }
```

Comment:
Check to see if there are any properties requested, add a debug level message if there were not, in case the user wants to set debug level logging to see if anything surprising happened here.